src/Crate/PDO/PDOStatement.php

```diff
@@ -202,7 +202,10 @@ public function execute($input_parameters = null)
             if (is_int($parameter) && $zero_based) {
                 $parameter++;
             }
-            $this->bindValue($parameter, $value);
+            // to correctly save arrays and objects, we need to pass
```

Comment:
Note that I'm just saying that `$stmt->execute($arrayOfValues);` should not be executed with typed parameters.

If you have parameter types, then loop and use `bindValue`, because the PDO API simply doesn't expose something "safe to use" anyway.